modules/store/schematics/ng-add/index.spec.ts

```diff
@@ -131,14 +131,16 @@ describe('Store ng-add Schematic', () => {
     expect(content).toMatch(/export interface AppState {/);
   });
 
-  it('should add runtime checks by default', () => {
-    const options = { ...defaultOptions, module: 'app.module.ts' };
+  it('should not add runtime checks by default', () => {
+    const options = {
+      ...defaultOptions,
+      module: 'app.module.ts',
+      minimal: true,
```

Comment:
Why is this added?

Reply:
to check if schematics creating the correct StoreModule.forRoot statement?
Maybe rename the test so it makes more sense ?

Reply:
I was pointing at the `minimal` flag, because it wasn't there before.
But I think this test can be removed, it was added to verify the immutability flags were added.